### PR TITLE
Implement corpus specific settings through corpus_config

### PR DESCRIPTION
**Pull request by BeritJanssen**
This branch tries to solve multiple problems:
- the number of arguments for the training script was growing out of hand
- training settings for specific corpora could not be retrieved after the fact
- it was not possible to modify the Elasticsearch queries

Some possible concerns with the current approach:
- while the dictionary approach has the advantage that it's easy to retrieve and check keys, it's also easy to mistype. Exceptions and warnings should take care of the necessary settings, but if you mistype `"vector_szie": 1000`, the setting will be ignored silently
- disclosing index names: not problematic for ES clusters which have enabled security features, but may be problematic for some reuse scenarios

### CODE REVIEW

**Review by JeltevanBoheemen — APPROVED** [review]
The idea is well implemented. 
I'm not super enthousiastic about including config in Python source code. I think this repo is an excellent candidate for a config file that can be supplied via commandline argument. For example, a Yaml file in some specified format. We could archive the used configurations in a private location, satisfying both an advantage and disadvantage of this solution.

That would constitute a bit more work though. We can go with this approach.

**Comment by BeritJanssen** [comment]
For most settings, it's actually an advantage to expose them through source control. It can then also serve as documentation for researchers who would like to know with which settings our word models were trained. The only potentially sensitive information, the name of the index, may actually be supplied through an .env file instead, comparable to the Elasticsearch values. Will think about this.
